new-evaluation.css

```diff
@@ -21,60 +58,57 @@
 			vertical-align: middle;
 		}
 
-		.div_form {
+		.evaluation_form {
 			padding-top: 55px;
 			margin: 0 auto ;
 			width: 1000px;
 			height: 40px;
 		}
 
-		.input {
+		.evaluation_form_header {
 			border: none;
 			border-bottom: 3px solid #f7f7f7;
 			height: 30px;
 			width: 300px;
 			margin: 7px;
 		}
 
-		.table {
+		.evaluation_form_table {
 			border-collapse: collapse;
 			width: 1000px;
 			margin: 20.750px 0px;
 			text-align: center;
 		}
-		
-		th {
-			background-color: #419fe2;
+
+		.evaluation_form_table_header {
+			background-color: #21bcff;
 			height: 40px;
 			color: white;
-			font-family: Arial;
 			font-size: 14px;
 			width: 935px;
-			
 		}
 
-		.tr {
+		.evaluation_form_table_content {
 			border: 1px solid black;
 			height: 40px;
 			width: 934px;
 		}
 
-		a {
-			margin-left: 100px;
-			margin-right: 15px;
-			font: 25px Arial;
-		}
-
-		a:link {
-			text-decoration: none;
-			color: black;
-		}
-
-		a:visited {
-			text-decoration: none;
-			color: black;
+		.evaluation_form_submit {
```

Comment:
Thanks! 😄 After these training days and after seeing how cleaner and better things can get, I'm starting to like it too.